spec/services/word_watcher_spec.rb

```diff
@@ -178,4 +178,15 @@
     end
   end
 
+  describe "apply_to_text" do
```

Comment:
Since this is a class method, we should probably prefix it with `.` like this: `describe ".apply_to_text(text)"`.

Reply:
Just to confirm but I think it should be `describe ".apply_to_text"` only without the need to specify the arguments to the method?

Reply:
That’s up to us I think :grin: (lately I was used to put the arguments there, so it’s just a habit :sweat_smile:)